docs/usage/guides/cuda.md

```diff
@@ -257,3 +341,4 @@ Most of the information in this article was obtained from various sources:
 * [Add NVIDIA GPU support to k3s with containerd](https://dev.to/mweibel/add-nvidia-gpu-support-to-k3s-with-containerd-4j17)
 * [microk8s](https://github.com/ubuntu/microk8s)
 * [K3S](https://github.com/rancher/k3s)
+* [k3s-gpu](https://gitlab.com/vainkop1/k3s-gpu)
```

Comment:
Let's add an authors section here so people know who to refer to for follow-up questions in the future :+1: 

```suggestion
* [k3s-gpu](https://gitlab.com/vainkop1/k3s-gpu)

## Authors

- [@markrexwinkel](https://github.com/markrexwinkel)
- [@vainkop](https://github.com/vainkop)
```